core/services/ocr2/plugins/ocr2keeper/evm/registry.go

```diff
@@ -614,15 +608,38 @@ func (r *EvmRegistry) doCheck(ctx context.Context, mercuryEnabled bool, keys []t
 	}
 }
 
+func splitKey(key ocr2keepers.UpkeepKey) (*big.Int, *big.Int, error) {
```

Comment:
Should this function live in the keepers repo, on the UpkeepKey type?

Reply:
Not on the UpkeepKey type since the goal was to push all that functionality to the encoder, but ideally it needs to use the encoder here instead.